package.json

```diff
@@ -48,6 +51,8 @@
     "font-awesome": "^4.7.0",
     "golden-layout": "^1.5.9",
     "lodash": "^4.17.10",
+    "lodash-es": "^4.17.11",
```

Comment:
if i am understanding correctly, we don't need to add lodash-es, right?

Reply:
Yes